pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -240,7 +240,7 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 	doc.OpenShiftCluster.Properties.ClusterProfile.PullSecret = ""
 	doc.OpenShiftCluster.Properties.ServicePrincipalProfile.ClientSecret = ""
 
-	b, err := json.MarshalIndent(converter.ToExternal(doc.OpenShiftCluster), "", "    ")
+	b, err := json.MarshalIndent(converter.ToExternal(doc.OpenShiftCluster), "", "\t")
```

Comment:
I doubt anything cares, but why change the indentation to tabs vs spaces?

Reply:
it's just a bit more readable imo , can revert it though, don't really care about it